roles/ceph-infra/tasks/configure_firewall.yml

```diff
@@ -18,17 +18,15 @@
       state: started
       enabled: yes
 
-  - name: open monitor and manager ports
+  - name: open monitor ports
     firewalld:
-      service: "{{ item.service }}"
-      zone: "{{ item.zone }}"
-      source: "{{ public_network }}"
+      service: "{{ ceph-mon }}"
```

Comment:
`ceph-mon` is a static value not a variable.
Also you shouldn't remove the `ceph` service from this task otherwise the ports for the mgr won't be opened when not using the dedicated mgr ansible group (they are collocated on the mon nodes in that case).

Reply:
Thanks for the guidance @dsavineau.

I hope I got the `with_nested` thing right.

I also changed the `open osd ports` task to use the list `union` filter so that we have just one task for both `cluster_network` and `public_network`.